src/codeu/chat/server/LocalFile.java

```diff
@@ -0,0 +1,170 @@
+package codeu.chat.server;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.Serializable;
+import java.io.ObjectOutputStream;
+import java.io.RandomAccessFile;
+import java.io.FileOutputStream;
+import java.io.ObjectInputStream;
+import java.io.FileInputStream;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import codeu.chat.common.ConversationHeader;
+import codeu.chat.common.Message;
+import codeu.chat.common.User;
+import codeu.chat.util.Logger;
+import codeu.chat.util.Uuid;
+
+public class LocalFile implements Serializable
```

Comment:
I don't believe this class needs to implement Serializable, since it's never serialized